provision.sh

```diff
@@ -27,6 +27,12 @@ wget -O- https://toolbelt.heroku.com/install-ubuntu.sh | sh
 echo "--------------- Installing both virtualenv and virtualenvwrapper ---"
 pip install virtualenvwrapper virtualenv
 
+# install postgresql and setup user
+echo "--------------- Installing postgresql ------------------------------"
+apt-get install -y python-dev libpq-dev postgresql postgresql-contrib
+su - postgres -c "createuser -s vagrant"
+su - vagrant -c "createdb $DB"
```

Comment:
While writing shell scripts, it's a good practice to wrap your variable names within braces. Otherwise, `$1` and `$11` would be indistinguishable. But if you clearly write `${1}` and `${11}`, then the shell interpreter knows what you are referring to, in an unambiguous way.

It's just good practice.